integration-test/src/features/getFeesMulti.feature

```diff
@@ -0,0 +1,263 @@
+Feature: GetFees - Get List of fees by CI, amount, method, touchpoint
+
+  Background: 
+    Given the configuration "dataCart.json"
+
+  Scenario: Execute a GetFeesMulti request
+    Given initial json
+      """
+      {
+        "bin": "309500",
+        "paymentMethod": "CP",
+        "touchpoint": "CHECKOUT",
+        "idPspList": null,
+        "paymentNotice": [
+            {
+                "primaryCreditorInstitution": "77777777777",
+                "paymentAmount": 899999999999999,
+                "transferList": [
+                    {
+                        "creditorInstitution": "77777777777",
+                        "transferCategory": "TAX1"
+                    }
+                ]
+            }
+        ]
+      }
+      """
+    When the client send a V2 POST to /fees
+    Then check statusCode is 200
+    And the body response for the bundleOptions.idsCiBundle field is:
+    | idCiBundle  |
+    | "int-test-1"  |
+    And the sum of the fees is correct and the EC codes are:
+    | feeCode  |
+    | "77777777777"  |
+
+  Scenario: Execute a GetFeesMulti request 2
```

Comment:
replace `Execute a GetFeesMulti request 2` with more context.
what changes from request 1 rather than request 3

Reply:
should be useful something stuffs as following : 
```
1. La commissione è maggiore della somma delle fee

2. La commissione è minore della somma delle fee

3. Non sono presenti ciBundles

4. Sono presenti più ciBundles ma un paymentNoticeItem ha EC code non corrispondente (nessuno selezionato)

5. Sono presenti più ciBundles ma un solo paymentNoticeItem (solo uno selezionato)

6. Sono presenti più ciBundles e più paymentNoticeItem

7. Sono presenti più ciBundles con più attributi, viene tornato il prodotto cartesiano
```